### ISSUE TITLE
[K3][Quest] Opening the Backdoor – available too early

### ISSUE THREAD

**Issue by Greywolf59** [WoWManiaUK/Redemption#6139]
[Opening the Backdoor](https://www.wow-mania.com/armory/?quest=12821) is available right away.
A: It should be available only as a third quest in chain:
[Expression of Gratitude](https://www.wow-mania.com/armory/?quest=12836) + [Reclaimed Rations](https://www.wow-mania.com/armory/?quest=12827) → [Ample Inspiration](https://www.wow-mania.com/armory/?quest=12828) → [Opening the Backdoor](https://www.wow-mania.com/armory/?quest=12821)
Based on:
https://wowwiki-archive.fandom.com/wiki/Quest:Opening_the_Backdoor#Quest_progression
https://wowpedia.fandom.com/wiki/Cell_Block_Tango#Progression (quest renamed, Blizz is no fun)
https://www.wowhead.com/wotlk/quest=12828/ample-inspiration#comments:id=532855

B: after [Clean Up](https://www.wow-mania.com/armory/?quest=12818) chain is completed:
https://www.wowhead.com/wotlk/quest=12828/ample-inspiration#comments:id=620426
[Clean Up](https://www.wow-mania.com/armory/?quest=12818) → [Just Around the Corner](https://www.wow-mania.com/armory/?quest=12819) → [Slightly Unstable](https://www.wow-mania.com/armory/?quest=12826) → [A Delicate Touch](https://www.wow-mania.com/armory/?quest=12820)
Based on:
https://www.wowhead.com/wotlk/quest=12828/ample-inspiration#comments:id=620426
https://wowpedia.fandom.com/wiki/Clean_Up#Progression
https://www.wowhead.com/wotlk/quest=12820/a-delicate-touch#comments:id=458109

C: after you save the goblin miner:
[Only Partly Forgotten](https://www.wow-mania.com/armory/?quest=12831) → [Bitter Departure](https://www.wow-mania.com/armory/?quest=12832)
Based on:
https://www.wowhead.com/wotlk/quest=12832/bitter-departure#comments:id=620477
https://www.wowhead.com/wotlk/quest=12832/bitter-departure#comments:id=568498

So in short:  Ample Inspiration + A Delicate Touch + Bitter Departure = → Opening the Backdoor

**Comment by Echinaceax** [comment]
this questline has been worked on before here @Greywolf59 https://github.com/WoWManiaUK/Redemption/issues/5736

**Comment by Lumber-Jack-2** [comment]
added exactly like that: 
Ample Inspiration + A Delicate Touch + Bitter Departure

once those 3 are rewarded, only then Opening the Backdoor is available. 

https://www.wowhead.com/quest=12821/cell-block-tango#comments:id=695989 this comment confirms that the issue report is correct
